website/common/bannedSlurs.js

```diff
@@ -0,0 +1,3 @@
+let bannedSlurs = ['TEST_PLACEHOLDER_SLUR_WORD_HERE'];
```

Comment:
since this code is only used server side it should go in website/server not website/common, otherwise it'll be included in the client too